plugins/chat/assets/javascripts/discourse/components/channels-list-public.gjs

```diff
@@ -46,8 +54,8 @@ export default class ChannelsListPublic extends Component {
   }
 
   get isThreadEnabledInAnyChannel() {
-    return this.currentUser?.chat_channels?.public_channels?.some(
-      (channel) => channel.threading_enabled
+    return this.chatChannelsManager.publicMessageChannels?.some(
```

Comment:
Small nitpick, but I think we can probably move this to `chatChannelsManager.hasThreadedChannels` or similar to prevent repeating this in several places.

Reply:
yes good idea, will do this